multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -24,6 +24,10 @@ externalDNSName: {{.ExternalDNSName}}
 # it for you.
 #hostedZone: ""
 
+# The ID of hosted zone to add the externalDNSName to.
+# Only necessary if there are multiple hosted zones with the same name
+#hostedZoneId: ""
```

Comment:
I'd say no for two reasons:
* We'd break a released API, though we are pre-1.0
* In most cases, I would say that there won't be two hosted zones with the same name on a single AWS account. I think it's reasonable to save 9/10 people a minute each at the cost of making things every so slightly more complicated for the 1/10. 

I could be very wrong about the second point, in which case we should break the API and just use the ID.